tomcat/src/main/java/org/apache/coyote/http11/ContentType.java

```diff
@@ -0,0 +1,31 @@
+package org.apache.coyote.http11;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum ContentType {
+
+    TEXT_HTML(List.of("html"), "text/html;charset=utf-8"),
+    TEXT_CSS(List.of("css"), "text/css"),
+    TEXT_JAVASCRIPT(List.of("js"), "text/javascript"),
+    TEXT_PLAIN(List.of("txt"), "text/plain"),
+    IMAGE_SVG_XML(List.of("svg"), "image/svg+xml");
+
+    private final List<String> fileExtensions;
+    private final String mimeType;
+
+    ContentType(List<String> fileExtensions, String mimeType) {
+        this.fileExtensions = fileExtensions;
+        this.mimeType = mimeType;
+    }
+
+    public static ContentType of(String fileExtension) {
+        return Arrays.stream(ContentType.values())
+                .filter(type -> type.fileExtensions.contains(fileExtension))
+                .findFirst().orElseThrow(() -> new RuntimeException("잘못된 파일 확장자입니다."));
```

Comment:
```suggestion
.findFirst().orElseThrow(() -> new RuntimeException("잘못된 파일 확장자입니다. :" + fileExtension));
```

사소한 부분이긴 한데, 잘못된 파일 확장자 입니다. 하고 뒤에 어떤 값이 문제인지 파악하기 쉽게 fileExtension도 같이 출력해주면 나중에 로그볼 때 훨씬 디버깅이 편할 것 같아요

Reply:
오 좋습니다 👍 반영했어요!
[refactor: 잘못된 파일 확장자 예외 메시지에 원인(확장자) 명시](https://github.com/woowacourse/java-http/pull/831/commits/a24915278402ad2e8cc00358a179862d973bb171)